doc/source/cluster/vms/user-guides/launching-clusters/index.rst

```diff
@@ -1,9 +1,9 @@
 .. _launching-vm-clusters:
 
-Launching Ray Clusters
-======================
+Launching Ray Clusters in the Cloud (AWS, GCP, Azure) or On-Premise
```

Comment:
Nits: 

Should be either On-Premises for the full word or On-Prem for the abbreviation.

Does the title appear too long in the doc sidebar, or is it ok? I haven't taken a look at the rendered docs.
If it needs to be shortened, "Launching" is redundant and "Ray" might be redundant -- one or both words could potential be removed.
On-Prem in place of the full word On-Premises would also keep it shorter.

Reply:
How about:

Ray Clusters on AWS / GCP / Azure / On-Prem

Reply:
sidebar looks ok. I changed to `Launching Ray Clusters on AWS, GCP, Azure, On-Prem` to be consistent with sub-page titles.